src/style.css

```
@@ -1 +1,46 @@
 /* add styles here */
+
+body {
+  margin: 0;
+  height: 96px;
+  width: 96px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star :not(:last-child) {
+  margin-right: 4px;
+}
+
+
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

you already have this styles in `.stars__star`, just set a new background-image
```suggestion
.stars--5 :nth-child(-n+5) {
  background-image: url(images/star-active.svg);
}
```